src/main/java/katas/Kata7.java

```diff
@@ -19,6 +21,19 @@
     public static List<Map> execute() {
         List<MovieList> movieLists = DataUtil.getMovieLists();
 
-        return ImmutableList.of(ImmutableMap.of("id", 5, "title", "Bad Boys", "boxart", "url"));
+        //return ImmutableList.of(ImmutableMap.of("id", 5, "title", "Bad Boys", "boxart", "url"));
+        return DataUtil.getMovieLists()
+                .stream()
+                .flatMap(movieList -> movieList.getVideos()
+                        .stream()
+                        .map(movie -> ImmutableMap.of(
+                                "id", movie.getId(),
+                                "title", movie.getTitle(),
+                                "boxart", movie.getBoxarts()
+                                        .stream()
+                                        .reduce((x1, x2) -> (x1.getHeight() * x1.getWidth()) > (x2.getHeight() * x2.getWidth()) ? x1 : x2)
+                                        .map(BoxArt::getUrl)
+                                        .get())))
```

Comment:
[mf] pls extract this huge block of code into separate private methods for better readability